src/main/java/cms/gov/madie/measure/services/TestCaseSequenceService.java

```diff
@@ -40,4 +40,22 @@ public void resetSequence(String measureId) {
           log.info("Reset sequence for test cases for measure [{}]", measureId);
         });
   }
+
+  public void setSequence(String measureId, int sequenceNumber) {
```

Comment:
It will be great to have some unit tests for this method, but if it is needed for release, may be we can keep in mind and do it for future releases

Reply:
@RohitKandimalla I tested from front end it works.